.github/workflows/lint.yaml

```
@@ -27,3 +21,7 @@ jobs:
       - name: Linting YAML files
         run: |
           gulp lintYaml
+
+      - name: Linting JavaScript files
+        run: |
+          gulp lintNode
```

Nit: for single-line run commands you can just put them in the same line:

```suggestion
      - name: Linting YAML files
        run: gulp lintYaml

      - name: Linting JavaScript files
        run: gulp lintNode
```